devops/jobs/MinosLifecycle.groovy

```diff
@@ -87,6 +88,7 @@ class MinosLifecycle {
                         env('ENVIRONMENT', environment)
                         env('DEPLOYMENT', deployment)
                         env('AWS_REGION',extraVars.get('AWS_REGION','us-east-1'))
+                        env('SNITCH', inner_config.get('snitch', ''))
```

Comment:
If this is empty, curl will fail in the bash script.  It seems like we should add an assert to ensure the snitch url is set?